Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -153,31 +150,29 @@ def test_cohesity_close_helios_incident(self):
             self.resource_group, self.workspace_name
         )
 
-        alert_details = get_alert_details(alert_id, self.api_key)
```

Comment:
Please keep the validations as is. Playbook run success doesn't mean that we are performing the action that the playbook is supposed to perform. An 'empty' playbook will return success as well. We have to perform the action validation.

Reply:
just get back the original validation.